maven/plugins/common/src/main/java/com/vmware/pscoe/maven/plugins/AbstractInstallNodeDepsMojo.java

```diff
@@ -80,10 +83,41 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 				.execute(getLog());
         }
 
-        new ProcessExecutor()
-			.name("Dependency installation")
-			.directory(project.getBasedir())
-			.command(deps)
-			.execute(getLog());
+        getLog().info("deps length  " + deps.stream().mapToInt(String:: length).sum());
+
+		for(String path : deps)
+		{
+			commandLenght = commandLenght + path.length();
+			if(commandLenght <= 7000)
+			{
+				deps_cmd_7000.add(path);
+			}
+			else
+			{
+				
+				new ProcessExecutor()
```

Comment:
If we do it like this, we will ignore the logic in:

```java
 if (!getLog().isDebugEnabled()) {
    deps.add("--silent");
}
```

Meaning that one of the executions may not be "silent"

Reply:
Hey @Michaelpalacce, added the missing logic in the depsCmdMax7000 list, and kept it in the original place as well, for the <= 7000 length case. Can you check if this is what you meant?